wsgi/openshift/services/models.py

```diff
@@ -1,8 +1,35 @@
 from django.db import models
+import json
+import requests
+import hashlib
 
 # Create your models here.
 
+class Location(models.Model):
+    ip = models.CharField(max_length=32, unique=True)
+    city = models.CharField(max_length=32)
+    region = models.CharField(max_length=32)
+    country = models.CharField(max_length=32)
+    latitude = models.CharField(max_length=32)
+    longitude = models.CharField(max_length=32)
 
+    def __unicode__(self):
+        return "IP: "+ self.ip + " City/Region/Country: " + self.city + "/" \
+        + self.region + "/" + self.country
+
+    def create(self, ip):
+        jsonData = requests.get("http://ipinfo.io/%s/json/" % ip).content
+        apiReturn = json.loads(jsonData)
+        longitude = apiReturn["loc"][0:7]
```

Comment:
Added `longitude, latitude = apiReturn['loc'].strip().split(',')` to strip out whitespace.